sw/device/lib/dif/dif_otbn.h

```diff
@@ -269,8 +269,8 @@ dif_otbn_result_t dif_otbn_get_err_code(const dif_otbn_t *otbn,
  * invalid, `kDifOtbnOk` otherwise.
  */
 dif_otbn_result_t dif_otbn_imem_write(const dif_otbn_t *otbn,
-                                      uint32_t offset_bytes,
-                                      const uint32_t *src, size_t len_bytes);
+                                      uint32_t offset_bytes, const void *src,
+                                      size_t len_bytes);
```

Comment:
These changes are fine - the mmio memcpy don't have alignment requirements, and use void* anyway, so this is all ok.